addons/web_kanban/static/src/less/kanban_dashboard.less

```diff
@@ -44,7 +44,6 @@
         }
 
         a {
-            color: @odoo-brand-optional;
```

Comment:
I have made this change because we have already done a code in less for font color for every kanban color but because of this code those font color didn't apply on fonts. So, i have removed this line and now all font colors are working properly.....thanx....